### PR TITLE
a tiny js world

### PR DESCRIPTION
**Pull request by drewzag**
### A Tiny JS World

[Demo](https://drewzag.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/drewzag/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/drewzag/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
+}
+
+// ======== OUTPUT ========
+
+const inhabitants = [dog, cat, woman, man, catWoman]
+
+const printInhabitants = (inhabitants) => Object.values(inhabitants).join('; ')
```
Comment (artem-trubin) [inline submissions/drewzag/a-tiny-js-world/index.js:59]: Doesn't this function receive 1 inhabitant?

Comment (artem-trubin) [inline submissions/drewzag/a-tiny-js-world/index.js:59]: The task specifically states not to use methods like `Object.values` and other similar ones.

File: submissions/drewzag/a-tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  saying: 'Dobriy den, everybody!',
+}
+
+const catWoman = {
+  species: 'cat-human',
+  name: 'Cat Marry',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
```
Comment (artem-trubin) [inline submissions/drewzag/a-tiny-js-world/index.js:52]: If we change `cat.saying`, would `catWoman.saying` change too? The task requires it to do so.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Hey, nice work, let's improve it. Check the comments below.

File: submissions/drewzag/a-tiny-js-world/index.js (hunk 2 above)
Comment (drewzag) [reply submissions/drewzag/a-tiny-js-world/index.js:52]: honestly, I think yes, it would change)

**Comment by drewzag** [comment]
@artem-trubin thanks for review)

File: submissions/drewzag/a-tiny-js-world/index.js (hunk 2 above)
Comment (drewzag) [reply submissions/drewzag/a-tiny-js-world/index.js]: Oh, sorry) I totally forget about getter and setter

**Review by artem-trubin — APPROVED** [review]
That's one way of doing this, good job! You could've also used `bind` method or connected them with prototyping.

**Pull request merged**